web/ui_utils.js

```diff
@@ -677,6 +677,13 @@ const animationStarted = new Promise(function (resolve) {
   window.requestAnimationFrame(resolve);
 });
 
+const docStyle =
+  typeof PDFJSDev !== "undefined" &&
+  PDFJSDev.test("LIB") &&
+  typeof document === "undefined"
+    ? null
```

Comment:
Why do we need this `null` value and the associated checks? If it's `null` the code that uses this should break AFAICT given that we don't check for `null` there. Moreover, even in the existing code we don't check if `document.documentElement.style` is actually defined, so is it ever undefined wouldn't it have been broken already?

Reply:
This is only done to prevent run-time errors when the unit-tests are run in Node.js, as is done with GitHub Actions.
Note the similar code just above: https://github.com/mozilla/pdf.js/blob/5b02c685d69cca319a1d35d0cf63bb1c733d6a5c/web/ui_utils.js#L666-L676

Reply:
Right, that makes sense. Thanks!